src/main/java/io/netty/incubator/channel/uring/Native.java

```diff
@@ -34,6 +34,7 @@
     static final int DEFAULT_RING_SIZE = Math.max(64, SystemPropertyUtil.getInt("io.netty.uring.ringSize", 4096));
     static final int DEFAULT_IOSEQ_ASYNC_THRESHOLD =
             Math.max(0, SystemPropertyUtil.getInt("io.netty.uring.iosqeAsyncThreshold", 25));
+    static final int DEFAULT_RING_FLAGS = Math.max(0, SystemPropertyUtil.getInt("io.netty.uring.ringFlags", 0));
```

Comment:
some flags have some logic changes in the implementation, for example sqpoll you don't have to submit anymore when IORING_SETUP_SQPOLL enable, I would prefer to have booleans as flags :) it's also easier the for the user to combine two flags, like that IORING_SETUP_SQPOLL | IORING_SETUP_SQ_AFF

Reply:
I am not sure I understand you.
You can do `IORING_SETUP_SQPOLL | IORING_SETUP_SQ_AFF` with integers:
```
$ jshell
|  Welcome to JShell -- Version 15.0.2
|  For an introduction type: /help intro

jshell> 1 | 2;
$1 ==> 3
```

Reply:
tbh I'm not huge fan to expose the flag(it's just my opionion) and it would be confusing for some people, it would be easier to understand when we use boolean variables instead :)